src/ui/components/delegation/DelegateCard.tsx

```diff
@@ -49,14 +50,22 @@ function extractRole(address: string, state: State): DelegateRoleType[] {
   if (state.type == 'ConnectedState') {
     if (state.chain.fellows.has(address)) {
       return ['fellow'];
+    } else {
+      // ToDo: remove (this is a mock role for UX testing)
```

Comment:
Don't we already have fellows part of the delegates on kusame? If not, I'd suggest we add some?

Reply:
They seem not be pulled in the state.delegates.